conf/messages.en

```diff
@@ -235,3 +235,14 @@ notLiable.think.title = Tell us what you think
 notLiable.think.info = This is a new service and we want to know what you think of it.
 notLiable.think.feedback = {0}. Your thoughts and ideas will help us to make improvements.
 notLiable.think.feedback.link.description = Send us your feedback
+
+liabilityProcessMoreWeightPage.sectionHeader = Liability details
+liabilityProcessMoreWeightPage.title = Do you expect to process more than 10,000kg of plastic packaging in the next 30 days?
```

Comment:
As mentioned previously, we probably should use the value from application config rather than hardcoding here.